packages/lockfile-lint-api/src/common/ParsingError.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const {LOCKFILE_TYPES} = require('./constants')
+
+const errorMessages = {
+  NO_OPTIONS: () => 'Did not recive options for lockfile path or type',
+  NO_PARSER_FOR_TYPE: type =>
+    `Unable to find relevant lockfile parser for type "${type}", the currently available options are ${LOCKFILE_TYPES}.`,
+  NO_PARSER_FOR_PATH: path =>
+    `Unable to find relevant lockfile parser for "${path}", consider passing the --type option.`,
+  READ_FAILED: path => `Unable to read lockfile "${path}"`,
+  PARSE_NPMLOCKFILE_FAILED: path => `Unable to parse npm lockfile "${path}"`,
+  PARSE_YARNLOCKFILE_FAILED: path => `Unable to parse yarn lockfile "${path}"`
+}
+
+const ERROR_KEYS = Object.keys(errorMessages).reduce(function (accumulator, key) {
+  return Object.assign(accumulator, {[key]: key})
+}, {})
```

Comment:
why not to just export `errorMessages` then?

Reply:
i was overcomplicating things. now changed. thanks!